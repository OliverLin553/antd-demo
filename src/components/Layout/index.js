import React from 'react';
import './style.css';

function Layout({ children }) {
  return (
    <header className="App-header">
      {children}
    </header>
  );
}

export default Layout;
