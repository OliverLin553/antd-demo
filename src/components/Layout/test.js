import React from 'react';
import { shallow } from 'enzyme';
import Layout from './';

describe('<Layout>', () => {
  it('it renders the expected tree', () => {
    const wrapper = shallow(<Layout />)
    expect(wrapper).toMatchSnapshot()
  })
})
