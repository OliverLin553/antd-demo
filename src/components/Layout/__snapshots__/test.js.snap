// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout> it renders the expected tree 1`] = `
<header
  className="App-header"
/>
`;
